sql/src/main/java/org/apache/druid/sql/calcite/parser/DruidSqlIngest.java

```diff
@@ -34,13 +34,18 @@
  */
 public abstract class DruidSqlIngest extends SqlInsert
 {
+  public static final String SQL_EXPORT_FILE_FORMAT = "__exportFileFormat";
```

Comment:
Is there validation to prevent user from setting up the field? I think we have some restricted context parameters that we can setup, and perhaps we should piggyback on that, or add our own validation (if there isn't). Also, a test to verify that.